docs/EN_US/ConfiguringHPCC/ConfiguringHPCC.xml

```diff
@@ -2500,13 +2640,35 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
                   xmlns:xi="http://www.w3.org/2001/XInclude" />
 
       <sect2>
-        <title></title>
+        <title />
 
         <sect3>
           <title>Additional Roxie Configuration items</title>
 
-          <para><emphasis role="bold">Add Servers to Roxie
-          Farm</emphasis></para>
+          <!--***Move to XSD for 2.0***-->
+
+          <para>
+            <emphasis role="bold">Persistent Connections to Roxie</emphasis>
+
+            
+          </para>
+
+          <para>Persistent connections can cause backwards compatibility
+          issues with non-standard HTTP clients. You can disable this feature
+          by setting <emphasis>maxHttpConnectionRequests</emphasis> to either
+          0 or 1.</para>
+
+          <para>In addition, persistent connections can affect Roxie load
+          balancing. If this is a concern, you can adjust
+          <emphasis>maxHttpConnectionRequests</emphasis> and
+          <emphasis>maxHttpKeepAliveWait</emphasis> values to optimize how
+          long all the transactions from a particular persistent connection go
```

Comment:
Makes it sound like we're able to affect the load balancer.  Rather something like:  "to optimize how many persistent connections a particular node will support, and how long they will last."